src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -60,7 +60,7 @@ def fixed_pos_embedding(x, seq_dim=1, seq_len=None):
     if seq_len is None:
         seq_len = x.shape[seq_dim]
     inv_freq = 1.0 / (10000 ** (torch.arange(0, dim, 2) / dim))
-    sinusoid_inp = torch.einsum("i , j -> i j", torch.arange(seq_len), inv_freq).to(x.device).float()
+    sinusoid_inp = torch.einsum("i , j -> i j", torch.arange(seq_len).float(), inv_freq).to(x.device).float()
```

Comment:
This point is important to review and to see if it doesn't break anything.

It was problematic because the `torch.arange` was int32 and the `inv_freq` was float.
It's not accepted by Onnx

Reply:
Thanks for flagging this change - it looks fine to me. Would the following be more idiomatic?

```suggestion
    sinusoid_inp = torch.einsum("i , j -> i j", torch.arange(seq_len, dtype=torch.float), inv_freq).to(x.device).float()
```